lib/jsdom/living/window/navigation.js

```diff
@@ -88,3 +102,80 @@ function urlEquals(a, b, flags) {
   }
   return flags.excludeFragments || a.fragment === b.fragment;
 }
+
+function loadPage(oldDocument, newURL) {
```

Comment:
A lot of this code looks like it's from frames code; can we remove the frames code now?